config/config.default.js

```diff
@@ -198,6 +199,9 @@ module.exports = appInfo => {
    */
   config.siteFile = {
     '/favicon.ico': fs.readFileSync(path.join(__dirname, 'favicon.png')),
+    option: {
+      cacheControl: 'public, max-age=2592000',
```

Comment:
Use `cacheControl` directly, siteFile is option config, no need to add a `option` for prefix.

```js
config.siteFile = {
  // default cache in 30 days
  cacheControl: 'public, max-age=2592000',
}
```